python/lsst/meas/extensions/gaap/_gaap.py

```diff
@@ -339,13 +377,16 @@ def measure(self, measRecord: lsst.afw.table.SourceRecord, exposure: afwImage.Ex
                 aperShape = afwGeom.Quadrupole(aperSigma2, aperSigma2, 0.0)
                 fluxResult = measBase.SdssShapeAlgorithm.computeFixedMomentsFlux(convolved.getMaskedImage(),
                                                                                  aperShape, center)
-                fluxScaling = sigma**2/aperSigma2  # Eq. A16 of Kuijken et al. (2015)
-
-                # Copy result to record
-                instFluxKey = measRecord.schema.join(baseName, "instFlux")
-                instFluxErrKey = measRecord.schema.join(baseName, "instFluxErr")
-                measRecord.set(instFluxKey, fluxScaling*fluxResult.instFlux)
-                measRecord.set(instFluxErrKey, fluxScaling*fluxResult.instFluxErr)
+                # Calculate the scale factors by which to scale the fluxes and
+                # their standard errors.
+                fluxScaling = 0.5*sigma**2/aperSigma2  # Eq. A16 of Kuijken et al. (2015)
+                fluxErrScaling = self._getFluxErrScaling(kernelAcf, aperShape)  # Eq. A17 of the same.
```

Comment:
Similarly here.  I think I need a lot more documentation steps in between A17 and what is being computed here.
I do see that there's more detail in the tests (which should be brought up to the code comments, not just in the tests).  However, I still can't follow what's being calculated.